tcms/rpc/serializer.py

```diff
@@ -428,7 +428,7 @@ class TestExecutionRPCSerializer(QuerySetBasedRPCSerializer):
     values_fields_mapping = {
         "id": ("id", do_nothing),
         "case_text_version": ("case_text_version", do_nothing),
-        "close_date": ("close_date", datetime_to_str),
+        "stop_date": ("stop_date", datetime_to_str),
```

Comment:
Remove this. The entire serializer module has been removed in the wip_remove_serializer branch.